### PR TITLE
REST module guide

### PR DESCRIPTION
**Pull request by Starzu**

### CODE REVIEW

**Comment by ddworak** [comment]
lgtm

File: guide/src/main/scala/io/udash/web/guide/views/rest/RestClientServerView.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    h2("REST interface usage"),
+    p(
+      "Once the REST API wrapper interface is prepared, you can create a REST connector based on it. ",
+      "The code below presents creation of an example connector: "
+    ),
+    CodeBlock(
+      """val restServer = DefaultServerREST[MainServerREST](
+        |  dom.window.location.hostname,
+        |  dom.window.location.port.toInt,
+        |  "/rest"
```
Comment (malcheg) [inline guide/src/main/scala/io/udash/web/guide/views/rest/RestClientServerView.scala:30]: `"/rest/"` or is it irrelevant?

**Comment by malcheg** [comment]
@Starzu looks good

File: guide/src/main/scala/io/udash/web/guide/views/rest/RestClientServerView.scala (hunk 1 above)
Comment (Starzu) [inline guide/src/main/scala/io/udash/web/guide/views/rest/RestClientServerView.scala:30]: It's irrelevant :)

**Pull request merged**